ExampleApp-macOS/ViewController.swift

```diff
@@ -14,7 +14,7 @@ class ViewController: NSViewController {
     fileprivate let attributedStrings: [NSAttributedString] = [
         {
             let attachment = TextAttachment()
-            attachment.image = NSImage(named: "Star")!
```

Comment:
The app is getting crash on this line because there is no such image. That's why I removed `!`.